app-config.yaml

```diff
@@ -17,6 +17,7 @@ backend:
     port: 7007
   csp:
     connect-src: [ "'self'", 'http:', 'https:' ]
+    img-src: [ '*' ]
```

Comment:
Letting this go for now as I need this for demo, but this needs to be self in future.